dotnet3.5/S3/DeleteBucketExample/DeleteBucket/DeleteBucket.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+// snippet-start:[S3.dotnet35.DeleteBucket]
+
+using Amazon;
+using Amazon.S3;
+using System;
+using System.Threading.Tasks;
+
+namespace DeleteBucket
+{
+    class DeleteBucket
+    {
+        // This example shows how to delete an existing and empty bucket.
+        //  The examples uses AWS .NET SDK 3.5 and .NET 5.0
```

Comment:
"AWS .NET SDK" -> "AWS SDK for .NET" -- here and in many other locations throughout.

Reply:
Yes, I searched for this throughout the projects and fixed it where it occurs.